index.js

```diff
@@ -95,60 +126,41 @@ const io = require('socket.io-client');
 const socket = io(config.socketioserver, {
   reconnect: true
 });
-socket.on('connect', function(socket) {
+socket.on('connect', () => {
   console.log('socket.io-client is connected!');
 });
 let lastChat = {};
-socket.on('lobby.chat', async function(response) {
-  let ChatDetail = new Date(response.timestamp);
-  let tempChatDate = ChatDetail.getDate();
-  let tempChatMonth = ChatDetail.getMonth() + 1;
-  let chatYear = ChatDetail.getFullYear();
-  let tempChatHours = ChatDetail.getHours();
-  let tempChatMinutes = ChatDetail.getMinutes();
-  let tempChatSeconds = ChatDetail.getSeconds();
-  let chatDate = (tempChatDate < 10) ? `0${tempChatDate}` : tempChatDate;
-  let chatMonth = (tempChatMonth < 10) ? `0${tempChatMonth}` : tempChatMonth;
-  let chatHours = (tempChatHours < 10) ? `0${tempChatHours}` : tempChatHours;
-  let chatMinutes = (tempChatMinutes < 10) ? `0${tempChatMinutes}` : tempChatMinutes;
-  let chatSeconds = (tempChatSeconds < 10) ? `0${tempChatSeconds}` : tempChatSeconds;
-
-  function getChatDate() {
-    return chatDate + "/" + chatMonth + "/" + chatYear +
-      " " + chatHours + ":" + chatMinutes + ":" + chatSeconds;
-  };
+socket.on('lobby.chat', async (response) => {
+  let ChatDate = datetime(response.timestamp);
 
   function getchatmsg() {
     return '``' + response.name + '``' + " : " + response.message;
   };
 
   function formatChatMessage() {
-    return '``' + getChatDate() + "`` " + getchatmsg();
+    return '``' + ChatDate + "`` " + getchatmsg();
```

Comment:
use back tick `  for concatenation strings

``return `'${ChatDate}' ${getchatmsg()}`;``